dist/angular-sticky-table-header.js

```diff
@@ -109,7 +109,8 @@ angular.module('turn/stickyTableHeader', ['watchDom']).value('stickyTableHeaderO
             } else if (scope.stuck && scrollY < scope.offset.top) {
               scope.setStuck(false);
             }
-            scope.clone.css('left', scope.offset.left - scrollX);
```

Comment:
this breaks horizontal scrolling, why did you remove this? or does abs positioning make it ok?